website/docs/language/stacks/deploy/link-stacks.mdx

```diff
@@ -0,0 +1,114 @@
+
+If the output value of a Stack changes after a run, HCP Terraform automatically triggers runs for any Stacks that depend on those outputs.
+
+## Background
+
+To output information from a Stack, declare a `publish_output` block in the deployment configuration of the Stack exporting data. We refer to the Stack that declares a `publish_output` block as the upstream Stack.
+
+To use another Stack's output, declare an `upstream_input` block in the deployment configuration of a different Stack in the same project. We refer to the Stack that declares an `upstream_input` block as the downstream Stack. For example, if Stack A produces outputs that Stack B depends on, Stack A is the upstream Stack, and Stack B is the downstream Stack.
+
+As a real life example, you could have a Stack for shared services, such as networking infrastructure, and a separate Stack for application components. Your Stack separation allows you to manage each Stack independently, and you can export data from your networking Stack with the `publish_output` block and consume that data into your application Stack using the `upstream_input` block.
+
+# Requirements
+
+The `publish_output` and `upstream_input` blocks require at least Terraform version `terraform_1.10.0-alpha20241009` or higher. We recommend downloading the [latest version of Terraform](https://releases.hashicorp.com/terraform/) to use the most up-to-date functionality.
+
+Downstream Stacks must also reside in the same project as their upstream Stacks.
+
+# Declare outputs
+
+You must declare a `publish_output` block in your deployment configuration for each value you want to output from your current Stack. 
+
+Once you apply a Stack configuration version that includes your `publish_output` block, HCP Terraform publishes a snapshot of those values, which allows HCP Terraform to resolve them. Meaning, you must apply your Stack’s deployment configuration before any downstream Stacks can reference your Stack's outputs.
+
+For example, you can add a `publish_output` block for the `vpc_id`  in your upstream Stack’s deployment configuration.
+
+<CodeBlockConfig filename="network.tfdeploy.hcl">
+
+```hcl
+# Networking Stack deployment configuration
+
+publish_output "vpc_id" {
+  description = "The networking Stack's VPC's ID."
+  # You can directly reference a deployment's values with the
+  # deployment.deployment_name syntax
+  value       = deployment.network.vpc_id
+}
+```
+
+</CodeBlockConfig>
+
+After applying this configuration, any Stack in the same project can now reference this `vpc_id` output by declaring an `upstream_input` block. Learn more about the [`publish_output` block](/terraform/language/stacks/reference/tfdeploy#publish_output-block-configuration).
+
+## Use an upstream Stack’s inputs
+
+Declare an `upstream_input` block in your Stack’s deployment configuration to read values from another Stack's `publish_output` block. Adding an `upstream_input` block creates a dependency on the upstream Stack.
```

Comment:
```suggestion
Declare an `upstream_input` block in your Stack’s deployment configuration to read values from another Stack's `publish_output` block. Configure the following arguments in the `upstream_input` block:

- `type`: Specifies the type of resource that the Stack imports data from.
- `source`: Specifies the path to the project containing the source resource.  

Adding an `upstream_input` block creates a dependency on the upstream Stack.
```
I'm totally just guessing based on the example.

Reply:
I don't think I need to add this level of field granularity to a usage article, folks can check out the reference on syntax which I link at the bottom of the section if they want all the details.